app/middleware/renderer.js

```diff
@@ -15,7 +18,36 @@ function age(req, res) {
 }
 
 function choose(req, res) {
-  res.render('choose');
+  return res.render('choose');
+}
+
+function location(req, res) {
+  if (!res.locals.correctLocationParams) {
+    return res.render('error');
```

Comment:
The `error` view contains the message about there having been a technical fault. As that isn't what has happened can this render a (possibly new) view informing the user they need to start over. Or, as I think has been agreed previously, can the start page be returned. However, if that does happen, there might need to be some way of informing the user what has happened.

Reply:
Suggest to send to the `start` page and have an error message displayed.